org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/colors/utils/ColorUtilsTest.java

```diff
@@ -0,0 +1,48 @@
+package org.eclipse.lemminx.extensions.colors.utils;
```

Comment:
Do you mind adding the copyright header that we use here? (You can list yourself as the contributor).